rust/astarte-message-hub-proto-mock/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# SPDX-License-Identifier: CC0-1.0
+
+[package]
+name = "astarte-message-hub-proto-mock"
+version = "0.1.0"
```

Comment:
I would add the version to the workspace to keep this in sync with the proto one